src/Integration.Vsix/AsmRef_Integration.Vsix_Baseline_WithStrongNames.txt

```diff
@@ -196,8 +196,8 @@ Relative path: 'SonarLint.VisualStudio.Education.dll'
 
 Referenced assemblies:
 - 'Microsoft.VisualStudio.ComponentModelHost, Version=17.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
-- 'Microsoft.VisualStudio.Interop, Version=17.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
```

Comment:
This `Microsoft.VisualStudio.Interop` is still referenced, but it's position in the file has changed.

I'm guessing the order changed in a merge. If I remember correctly, the AsmRef tool sorts the references into alphabetical order. The new version of the file is in sorted order.